content/push-gateway-api.md

```diff
@@ -39,6 +39,11 @@ notification provider (e.g. APNS, GCM).
 
 ## API standards
 
+### Standard error response
+
+All push gateway API endpoints MUST return error responses conforming to the
+[standard error response](/appendices#standard-error-response) schema.
```

Comment:
I'm unconvinced this is true; Synapse does not require the push gateway to return standard error responses, and I don't think Sygnal does so.

If we want to change this, I think it requires an MSC.